ocp_resources/image_content_source_policy.py

```diff
@@ -7,3 +7,28 @@ class ImageContentSourcePolicy(Resource):
     """
 
     api_group = Resource.ApiGroup.OPERATOR_OPENSHIFT_IO
+
+    def __init_(self, source=None, mirrors=None, **kwargs):
+        """
+        Create/Manage ICSP configuration object
+
+        Args:
+            source (string): the repository that users refer to, e.g. in image pull specifications.
+            mirrors (list): one or more repositories that may also contain the same images.
+                The order of mirrors in this list is treated as the user’s desired priority
```

Comment:
this allows adding only one policy; but there could be a few.
should get repositoryDigestMirrors (repository_digest_mirrors) which will get a list of dicts.

Reply:
done; let me know wyt